lib/Dialect/FIRRTL/Transforms/ExtractInstances.cpp

```diff
@@ -1175,9 +1186,10 @@ void ExtractInstancesPass::createSchema() {
   mlir::Type portsType[] = {
       StringType::get(context), // name
       PathType::get(context),   // extracted instance path
-      StringType::get(context)  // filename
+      StringType::get(context), // filename
```

Comment:
Please call get once and reuse the value.  Type uniquing has costs we don't need to pay here.